tests/modules/services/window-managers/sway/sway-default.nix

```diff
@@ -6,9 +6,7 @@ with lib;
   config = {
     wayland.windowManager.sway = {
       enable = true;
-      package = pkgs.runCommandLocal "dummy-package" { } "mkdir $out" // {
-        outPath = "@sway";
-      };
+      package = pkgs.writeScriptBin "sway" "" // { outPath = "@sway"; };
```

Comment:
Am I correct in thinking the tests are changed as they would now drag `sway`/`i3` into their closure?

Reply:
I think that was the reason for this change